src/server.c

```diff
@@ -4644,6 +4644,7 @@ int isReadyToShutdown(void) {
     listRewind(server.replicas, &li);
     while ((ln = listNext(&li)) != NULL) {
         client *replica = listNodeValue(ln);
+        if (replica->flag.protected_rdb_channel && replica->repl_data->rdb_client_disconnect_time) continue;
```

Comment:
In fact, can we just check repl_rdb_channel? That is, do we need to wait for the rdb replica client? ie. if a rdb client is receiving the RDB, should we wait it?